README.md

```diff
@@ -14,23 +14,21 @@ both the back-end server and their front-end app.
 2. Clone this [repository](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back.git) to that folder
 3. Create `.env` file in the root of backend folder (you can copy from `.env.example` and set it with your vars)
 4. Run `npm install` to install dependencies inside backend directory
-5. Run `npm run dev` to start the development server
-6. Open http://localhost:8000/api/v1/ with your browser to test
-7. Your back-end server is now running. You can now run the front-end app
-8. Swagger API Docs are available at http://localhost:8000/docs (json format at http://localhost:8000/docs.json)
+5. Run `prisma migrate dev` to apply migrations
```

Comment:
this requires adding prisma globally

Reply:
That's strange as `npm install prisma` adds a binary to node_modules/.bin:
<img width="272" alt="image" src="https://github.com/user-attachments/assets/5215cb36-1f40-4b0f-b3e3-111ca7c7ff6a" />